packages/components/radio/src/radio.js

```diff
@@ -0,0 +1,63 @@
+import { TSElement, unsafeCSS, html, customElementDefineHelper } from '@tradeshift/elements';
+import css from './radio.css';
+import { customEventNames } from './utils/constants';
+
+customElementDefineHelper(
+	'ts-radio',
+	class extends TSElement {
+		static get styles() {
+			return [TSElement.styles, unsafeCSS(css)];
+		}
+
+		static get properties() {
+			return {
+				value: { type: String, reflect: true },
+				label: { type: String, reflect: true },
+				checked: { type: Boolean, reflect: true },
+				disabled: { type: Boolean, reflect: true },
+				dir: { type: String, reflect: true }
+			};
+		}
+
+		handleClick(e) {
+			e.preventDefault();
+			e.stopPropagation();
+			this.dispatchEvent(new Event(customEventNames.RADIO_CLICK, { bubbles: true, composed: true }));
```

Comment:
Why not using the `this.dispatchCustomEvent`?

Reply:
Fixed